recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/0001-dts_dom0-Add-ranges-that-AUDMACs-relies-on.patch

```diff
@@ -0,0 +1,49 @@
+From f3174055c85d01b04dbd791d90562e017684b80c Mon Sep 17 00:00:00 2001
+From: Oleksandr Tyshchenko <olekstysh@gmail.com>
+Date: Tue, 24 Jan 2017 21:06:12 +0200
+Subject: [PATCH] dts_dom0: Add ranges that AUDMACs relies on
```

Comment:
Pls clarify patch subject -> change "dts_dom0" to something like "r8a7795(6).dtsi"

Reply:
done